03-postgres-typeorm/README.md

```diff
@@ -0,0 +1,96 @@
+# NestJS Boilerplate
```

Comment:
it would be great if we add the projects modules description like what this project have functionality which will really help other developer better understanding of how to use the code